recipe/meta.yaml

```diff
@@ -73,16 +73,16 @@ test:
 
 
 about:
-  home: https://github.com/Microsoft/LightGBM
+  home: https://github.com/microsoft/LightGBM
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: LightGBM is a gradient boosting framework that uses tree based learning algorithms.
 
   description: |
     A fast, distributed, high performance gradient boosting (GBDT, GBRT, GBM or MART) framework based on decision tree algorithms, used for ranking, classification and many other machine learning tasks.
-  doc_url: http://lightgbm.readthedocs.io/en/latest/
-  dev_url: https://github.com/Microsoft/LightGBM
+  doc_url: https://lightgbm.readthedocs.io/en/stable/
```

Comment:
I believe it's better to point users to the docs actual for the current release, not to the latest ones, because users can't get nightly builds from conda.

Refer to https://github.com/microsoft/LightGBM/issues/5930#issuecomment-1596621133.